internal/oci/uvm.go

```diff
@@ -182,6 +185,17 @@ func parseAnnotationsBool(ctx context.Context, a map[string]string, key string,
 	return def
 }
 
+// ParseAnnotationCommaSeparated searches `annotations` for `annotation` corresponding to a
+// list of comma separated strings
+func ParseAnnotationCommaSeparated(annotation string, annotations map[string]string) []string {
+	cs, ok := annotations[annotation]
+	if !ok || cs == "" {
+		return nil
+	}
+	results := strings.Split(cs, ",")
+	return results
```

Comment:
Can just 
```go
return strings.Split(cs, ",")
```

Reply:
Why did this highlight all these lines.. just referring to lines 195 and 196

Reply:
sry for confusion

Reply:
sure